reconcile/utils/terrascript_aws_client.py

```diff
@@ -837,7 +837,7 @@ def populate_additional_providers(self, accounts):
             alias = self.get_alias_name_from_assume_role(assume_role)
             ts = self.tss[account_name]
             config = self.configs[account_name]
-            existing_provider_aliases = [p.get("alias") for p in ts["provider"]["aws"]]
+            existing_provider_aliases = {p.get("alias") for p in ts["provider"]["aws"]}
```

Comment:
Why change it to a set? Memory concerns?

Reply:
faster lookup, O(N) for list, O(1) for set